jest.conf/testUtils.js

```diff
@@ -29,3 +22,56 @@ export const testAfterResize = testFunction => {
   };
   animationFrameId = requestAnimationFrame(assertAfterResize);
 };
+
+export async function renderComponent(page, component, props) {
+  await page.evaluate(
+    ({ component, props }) => {
+      window.postMessage(
+        {
+          type: 'RENDER_COMPONENT',
+          component: component,
+          props: props,
+        },
+        '*'
+      );
+    },
+    { component, props }
+  );
+  await page.waitForSelector('#testing-playground');
+
+  const isComponentRendered = await page.evaluate(() => {
+    const testing_playground = document.querySelector('#testing-playground');
```

Comment:
Just a note that in the future we should look for a more reliable way to know if the component was actually rendered.

Reply:
One thing we can do is capture the inital content of the testing playground page and compare it with the html content captured after rendering the component and run a check to ensure that they both are different.
Does it seem like a good option?

Reply:
Lets see how it works :)